app/views/static_pages/splash_page.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="layouts">
+  <div class="parent">
+      <div class="card" id="leftToRight">
+          <h2>Money Manager</h2>
+          <% if current_user %>
+            <p class="text-white">Welcome <%= current_user.name.capitalize %></p>
+            <%= link_to 'Groups', groups_path, class: 'btn btn-primary' %>
+            <%= link_to "Log out", destroy_user_session_path, data: { turbo_method: :delete }, class: 'btn btn-secondary'  %>
+          <% else %>
+          <div class="buttons">
+            <%= link_to "Log in", new_user_session_path, class: "btn btn-primary" %>
+            <%= link_to "Register", new_user_registration_path, class: "btn btn-secondary"  %>
+          </div>
+          <% end %>
+      </div>
+  </div>
+</div>
```

Comment:
- [ ] Kindly make sure the authentication page sticks as much as possible on the template design and you should use the correct **font** on your project title text 

| Template design | Your Design |
| ------------------------- | ---------------------- |
| ![Screenshot from 2022-10-21 17-55-32](https://user-images.githubusercontent.com/83042742/197239443-8d9b5720-ec3d-4726-9491-4e0fa23e258f.png) | ![Screenshot from 2022-10-21 18-02-30](https://user-images.githubusercontent.com/83042742/197239712-8ad7fb55-5c1b-43c0-8d08-ee02e303015c.png) |

Reply:
- [ ] This was not addressed